### DISCUSSION TITLE
Arbitrary year selector on login page

### DISCUSSION THREAD

**Discussion by Phiwa** [dani-garcia/vaultwarden#2639] [Q&A] [answered]
### Subject of the issue

There is some arbitrary year selector component on the login page.

### Deployment environment

Admin page does not work for me (see #2636).

* vaultwarden version: 1.25.1

* web-vault version: 2022.6.2

* Install method: Docker image

* Clients used: web vault (for this issue)

* Reverse proxy and version: Traefik 1.7

* MySQL/MariaDB or PostgreSQL version: SQLite

* Other relevant details: 
  * Environment variable "ADMIN_TOKEN" has been set and there is no config file which could override it.
  * Tested with the following browsers:
    * Chromium 89.0.4389.114
    * Microsoft Edge 103.0.1264.62

### Steps to reproduce

Open web vault page.

### Expected behaviour

No arbitrary year selector component. ;)

### Actual behaviour

Arbitrary year selector component. ;)

### Troubleshooting data

![Vaultwarden_date_picker_login_page](https://user-images.githubusercontent.com/1133120/180025206-d1c109cd-d267-4cf4-b112-189ecab09013.png)

**Comment by BlackDex** [comment, marked as answer]
See the comments on #2638
